protocol-units/settlement/mcr/contracts/src/staking/MovementStaking.sol

```diff
@@ -141,9 +132,25 @@ contract MovementStaking is
         uint256 epoch,
         address custodian,
         address attester,
+        address delegator,
         uint256 amount
     ) internal {
-        epochStakesByDomain[domain][epoch][custodian][attester] -= amount;
+        uint256 stake = epochStakesByDomain[domain][epoch][custodian][attester][
+            delegator
+        ];
+        stake -= amount; // okay to let this error if it underflows
+        epochStakesByDomain[domain][epoch][custodian][attester][
```

Comment:
funds get stuck if it underflows, better handle it with "if amount > _stake, _stake = 0"